### ISSUE TITLE
kaput files search --interactive

### ISSUE THREAD

**Issue by manicExpressive** [davidchalifoux/kaput-cli#65]
I'm a daily `kaput-cli` user and something I do a lot is searching for something in my put.io account so that I can download or delete it.

I'd find it really helpful to have an `--interactive` flag that would send the results of a search to an interface where I can then select one of the objects to do an action to it. My imagined workflow would be something like:

```
kaput files search --interactive weezer-live

> [ ] weezer-live-2021.mp4
> [*] weezer-live-2022.mp4

download
```

Note: This is deeply half baked and while typing this out I realized that I could use tmux's vi-copy-mode[1] to get the core functionality of "search and download file without touching mouse"  :D

1. https://waylonwalker.com/tmux-copy-mode/

**Comment by davidchalifoux** [comment]
I agree that this would be a much better UX compared to the current copy-and-paste workflow. I actually started the v2 rewrite with Go and [Bubbletea](https://github.com/charmbracelet/bubbletea) with the goal of making the entire program interactive, but I ended up pivoting to Rust for more of a personal challenge and because the non-interactive approach is more extensible for scripting on top of.

If I get the time for it I'll look into this, otherwise PRs are welcome!

**Comment by davidchalifoux** [comment]
This is available now in [v2.6.0](https://github.com/davidchalifoux/kaput-cli/releases/tag/v2.6.0) (#69)!

Thank you to @mpobrien for this.
